src/app/create-email-template/[templateId]/page.tsx

```diff
@@ -0,0 +1,21 @@
+import { CreateEmailTemplate } from '@forms/CreateEmailTemplate/CreateEmailTemplate';
+import { PageProps } from '@utils/types';
+import { getTemplate } from '@utils/form-actions';
+import { redirect, RedirectType } from 'next/navigation';
+import { validateEmailTemplate } from '@utils/validate-template';
+
+const CreateEmailTemplatePage = async ({
```

Comment:
Can we call this "edit" rather than "create".  One of the advantages of having these as separate pages is that we can have more descriptive URL paths.  Might be worth renaming `CreateEmailTemplate` to `EmailTemplateForm` or similar.